openerp/openupgrade/doc/source/modules80-90.rst

```diff
@@ -474,7 +474,7 @@ missing in the new release are marked with |del|.
 +----------------------------------------+------------------------------------------+
 | |del| purchase_analytic_plans          | Moved to OCA [#purchase_analytic_plans]_ |
 +----------------------------------------+------------------------------------------+
-| |del| purchase_double_validation       |                                          |
+| |del| purchase_double_validation       | Done                                     |
```

Comment:
Do we want to change this to `Done (merged into purchase)` or is that redundant?

Reply:
Yeah, I think that's the best thing!